src/Turbo/Resources/doc/index.rst

```diff
@@ -326,18 +326,23 @@ Let's discover how to use Turbo Streams to enhance your `Symfony forms`_::
                 // ... perform some action, such as saving the task to the database
 
                 // 🔥 The magic happens here! 🔥
-                if (TurboStreamResponse::STREAM_FORMAT === $request->getPreferredFormat()) {
+                if (TurboBundle::STREAM_FORMAT === $request->getPreferredFormat()) {
                     // If the request comes from Turbo, set the content type as text/vnd.turbo-stream.html and only send the HTML to update
-                    $request->setFormat(TurboStreamResponse::STREAM_FORMAT);
-                    return $this->render('task/success.stream.html.twig', ['task' => $task], new TurboStreamResponse());
+                    $request->setFormat(TurboBundle::STREAM_FORMAT);
+                    return $this->render('task/success.stream.html.twig', ['task' => $task]);
                 }
 
                 // If the client doesn't support JavaScript, or isn't using Turbo, the form still works as usual.
                 // Symfony UX Turbo is all about progressively enhancing your apps!
                 return $this->redirectToRoute('task_success', [], Response::HTTP_SEE_OTHER);
             }
 
-            // Symfony 5.3+
+  .. versionadded:: 2.1
+  
+  In versions prior to 2.1, TurboStreamResponse::STREAM_FORMAT was used instead of TurboBundle::STREAM_FORMAT. Also, one had to return a new TurboStreamResponse() object as the third argument for this->render().
```

Comment:
Let's move this above the code block, so people definitely see it.

Also, add manual line breaks around the 72nd character.

And put "ticks" (these characters: \`) around the technical words like `TurboStreamResponse::STREAM_FORMAT`.